htdocs/core/tpl/objectline_create.tpl.php

```diff
@@ -477,7 +477,7 @@ function price2numjs(num)
 
 /* JQuery for product free or predefined select */
 jQuery(document).ready(function() {
-	$("#prod_entry_mode_free").on( "click", function() {
+	$("#tablelines").on( "click", "#prod_entry_mode_free", function() {
```

Comment:
@eldy 
In fact it works with the original code, I confused with "live", which has been removed from jquery. I close this PR